CodeofConduct.md

```diff
@@ -0,0 +1,72 @@
+# 🌟 Code of Conduct
+
+## Our Pledge 🤝
+
+We as members, contributors, and leaders pledge to make participation in the ServiceNow Hacktoberfest community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
```

Comment:
I would either remove the word `Hacktoberfest` from this line, or  replace the word `community` with `event`. I.e.: either:
- ServiceNow community, or
- ServiceNow Hacktoberfest event

Reply:
changed to ServiceNow Hacktoberfest event